### PR TITLE
[4.1] Cassiopeia filter buttons

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
Give the filter buttons some love

### Testing Instructions
Edit an article on the front end and use any of the xtd-editor plugins eg Insert article
and
Test the versions button at the bottom

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

there is no change with the versions popup as the css was not used there

### CODE REVIEW

**Comment by richard67** [comment]
@brianteeman Have you checked also with RTL?

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on cb2d7a6c0bee97c65cccdb8ea8029460235653c8<br /><br />Filter buttons have equal distance now. Content history modal not changes. RTL also fine regarding these changes (other RTL glitches are unrelated).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37260">issues.joomla.org/tracker/joomla-cms/37260</a>.</sub>

**Comment by Quy** [comment]
37260-menu

File: build/media_source/templates/site/cassiopeia/scss/system/searchtools/searchtools.scss [hunk 1]
```diff
@@ -48,3 +48,9 @@
     margin-inline-start: 8px;
   }
 }
+
+.js-stools-field-selector {
+  .form-select {
+    width:auto;
```
Comment (richard67) [inline build/media_source/templates/site/cassiopeia/scss/system/searchtools/searchtools.scss]:
```suggestion
    width: auto;
```
Fix CS error in Drone

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on cbe89841677c790f2033c3ef8dcb4e72baa3b386<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37260">issues.joomla.org/tracker/joomla-cms/37260</a>.</sub>

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on cbe89841677c790f2033c3ef8dcb4e72baa3b386<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37260">issues.joomla.org/tracker/joomla-cms/37260</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37260">issues.joomla.org/tracker/joomla-cms/37260</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**